README.md

```diff
@@ -31,6 +31,29 @@ The Commerce App is a sample application that demonstrates the use of ListFeed p
 
 [Browse source](https://github.com/unoplatform/Uno.Samples/tree/master/reference/Commerce)
 
+### SimpleCalculator App
+
+The SimpleCalculator App is a sample application that is designed to perform basic arithmetic operations. The app was built using 4 options - [C# Markup](https://aka.platform.uno/csharp-markup) or [XAML]() for the UI and [MVUX](https://aka.platform.uno/mvux) or [MVVM]() for state management.
+
+[Browse source](https://github.com/unoplatform/Uno.Samples/tree/master/reference/SimpleCalc)
+
+### Counter App
+
+Discover the simplicity and power of Uno.Extensions with the Counter App, a straightforward yet powerful demonstration of basic and advanced features of the Uno Platform. This app provides a hands-on experience with fundamental concepts such as state management, user interaction, and UI updates in real-time.
```

Comment:
@vatsashah45, @kazo0, @nickrandolph 
I think we could use this simple aka link (https://aka.platform.uno/counter-tutorial) that gives the overview of the four variants of the Counter workshop.

So we can mention something additional like:
`This sample app was built using the four variants of the [Counter workshop](https://aka.platform.uno/counter-tutorial), combining markup language (XAML or C# Markup) and presentation framework (MVVM or MVUX).`